services/web/pkg/config/defaults/defaultconfig.go

```diff
@@ -98,7 +98,7 @@ func DefaultConfig() *config.Config {
 					ResponseType: "code",
 					Scope:        "openid profile email",
 				},
-				Apps: []string{"files", "search", "text-editor", "pdf-viewer", "external", "admin-settings", "epub-reader", "draw-io"},
+				Apps: []string{"files", "search", "text-editor", "pdf-viewer", "external", "admin-settings", "epub-reader", "draw-io", "ocm"},
```

Comment:
@kulmann can I keep `ocm` here? Or is it better to delete it. 
I need it for testing `ocm` on the web, but I don't need it for api tests

Reply:
I removed it and added info how enable ocm app in docs